src/js/pie.js

```diff
@@ -21,19 +21,25 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // 4. update the pie when the data set changes, including adding or removing slices
 
     fluid.defaults("floe.chartAuthoring.pieChart.pie", {
-        gradeNames: ["floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent",  "autoInit"],
+        gradeNames: ["floe.chartAuthoring.valueBinding","floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent"],
```

Comment:
@amb26 - what errors are you introducing? I get the following when inserting an indentation error and a loose equality vs. strict:

```
Linting src/js/pie.js ...ERROR
[L24:C5] W015: Expected 'gradeNames' to have an indentation at 9 instead at 5.
    gradeNames: ["floe.chartAuthoring.valueBinding","floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent"],
[L214:C37] W116: Expected '===' and instead saw '=='.
                return typeof (d) == "object" ? d.value : d;
```